common/src/main/java/org/apache/celeborn/common/write/PushState.java

```diff
@@ -67,8 +67,8 @@ public int nextBatchId() {
     return inFlightRequestTracker.nextBatchId();
   }
 
-  public void addBatch(int batchId, String hostAndPushPort) {
-    inFlightRequestTracker.addBatch(batchId, hostAndPushPort);
+  public void addBatch(int batchId, int batchBytesSize, String hostAndPushPort) {
```

Comment:
Please also update the invoker in `FlinkShuffleClientImpl`, then check the CI result.